torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1277,6 +1277,12 @@ def statically_known_true(x: Union[bool, SymBool]) -> bool:
     return x
 
 
+# a or b is eager, meaning that (a==1 or True) will specialize a.
+# but operator.or_(a==1, True) generates sympy expression and avoid
+# the specialization.
+sym_or = operator.or_
```

Comment:
this already exists on SymNode https://www.internalfb.com/code/fbsource/[6f1893cbc5666e46b21f179a39bea4d5df2fd462]/xplat/caffe2/torch/fx/experimental/sym_node.py?lines=439